eval.py

```diff
@@ -56,6 +57,13 @@ async def eval_(self, ctx, *, code):
         If her name was `eval`, no consequence happen in !eval <code>... 
         but errors can happen in other code in that file
         """
+        
+        # get only code without highlighter and code block
+        code_block = code.strip('`')
+        first_line = code_block.splitlines()[0]
+        if re.fullmatch(r'( |[0-9A-z]*)\b', first_line):
+            code = code_block[len(first_line)+1:]
+        
```

Comment:
The regex works well, but the code is not tested for some case ; by example : 
\`\`\`print("x")\`\`\` provide an error
\`\`\`
print("x")\`\`\` same

if the user simply want to use a block, and not a language.